### ISSUE TITLE
Improve build flow

### ISSUE THREAD

**Issue by ghost** [xivapi/vue-xivtooltips#3]

**Comment by ghost** [comment]
The Vue [documentation](https://vuejs.org/v2/cookbook/packaging-sfc-for-npm.html#How-do-I-configure-the-Rollup-build) has an example using rollup, but Ack says that the buble builder is... probably not good.

Vue cli 3 has a 'library' build mode, which seems promising (documented [here](https://cli.vuejs.org/guide/build-targets.html#library))

**Comment by ghost** [comment]
Promising results adding the vue build service (`yarn add -D @vue/cli-service @vue/cli-plugin-babel @vue/cli-plugin-eslint @vue/cli-plugin-vuex`), then adding a script in package.json:

```json
{
  "scripts": {
    "lint": "eslint --ext .js --ext .vue src",
    "docs": "vue-docgen -c docgen.config.js",
    "build": "cross-env NODE_ENV=production rollup -c",
    "build:dev": "rollup -c ",
    "vue:build": "vue-cli-service build --target lib --name VueXIVTooltips src/index.js"
  }
}
```

This results in 4 files being created:

```
VueXIVTooltips.common.js
VueXIVTooltips.common.js.map
VueXIVTooltips.css
VueXIVTooltips.umd.js
VueXIVTooltips.umd.js.map
VueXIVTooltips.umd.min.js
VueXIVTooltips.umd.min.js.map
```

**Comment by ghost** [comment]
Fixed by #32
